sopel/modules/search.py

```diff
@@ -130,8 +135,9 @@ def bing(bot, trigger):
         bot.reply("No results found for '%s'." % query)
 
 
-@commands('search')
-@example('.search sopel irc bot')
+@plugin.command('search')
+@plugin.example('.search sopel irc bot')
+@plugin.output_prefix(PLUGIN_OUTPUT_PREFIX)
 def search(bot, trigger):
     """Searches both Bing and DuckDuckGo."""
     if not trigger.group(2):
```

Comment:
I can't touch the line below; thanks for the artificial limitation, GitHub.

Up to you whether to use `help_prefix` (like I suggested for `.bing`) or rewrite the message so it doesn't use a command name (like I suggested for `.duck`).

Reply:
Yeah fixed it.